win32ss/user/ntuser/scrollbar.c

```diff
@@ -528,6 +528,11 @@ co_IntSetScrollInfo(PWND Window, INT nBar, LPCSCROLLINFO lpsi, BOOL bRedraw)
    Info = IntGetScrollInfoFromWindow(Window, nBar);
    pSBData = IntGetSBData(Window, nBar);
 
+   if (lpsi->fMask & SIF_THEMED && !(Info->fMask & SIF_THEMED))
```

Comment:
It looks like the second condition is somehow redundant.

Reply:
It prevents setting the flag when it is already set?

Reply:
Just a question: why is this flag not set for comctl32 treeview, but most likely is already set properly for comctl32 listview?
Wouldn't that actually show us the root-cause of the bug, instead of "fixing-up-later"?
I am just asking this in the hope, we would find something even more correct than your proposal.
Still it is by no means meant as a DoNotCommit, as your proposal is much better than all previous approaches.

Reply:
see jira.